DTS-STN/DECD-Design-System
# Ds 56/footer

Main Reviewer: @jvaddinen 

**Description & Background Context:**
Footer component based on https://www.figma.com/file/5fFU4nLH24E0wRdOk3amPS/DECD-Design---Starter-Figma-File?node-id=77%3A1657 designs and requirements

**Where should the reviewer start?**
File name: Footer folder (excluding static)

**How should this be tested?**
open storybook branch for component and check rendering on different screen sizes. Import component into a different project

**Are there any breaking changes?**
No
